src/coreclr/vm/amd64/asmhelpers.S

```diff
@@ -307,3 +307,44 @@ NESTED_ENTRY ProfileTailcallNaked, _TEXT, NoHandler
 
   ret
 NESTED_END ProfileTailcallNaked, _TEXT
+
+#ifdef TARGET_OSX
+# EXTERN_C void* GetThreadVarsSectionOffset()
+# <NOTE>
+# Helper to calculate the address of relevant __thread_vars section that holds the address of symbol tlv_get_address for thread
+# local `t_ThreadStatics`. The address is updated by the linker, which we retrieve here. In JIT code, this address is called
+# to retrieve the address of the thread local.
+# </NOTE>
+LEAF_ENTRY GetThreadVarsSectionOffset, _TEXT
+        mov     rdi,    _t_ThreadStatics@TLVP[rip]
+        ret
```

Comment:
```suggestion
        int 3
```
This will make it clear that this helper is not actually executed (Linux helper does the same thing).

Reply:
yep, did that, but was not pushed.